src/frontend/shared-components/src/__snapshots__/storyshots.test.js.snap

```diff
@@ -1866,6 +1866,463 @@ exports[`Storyshots Loader Page 1`] = `
 </Loader>
 `;
 
+exports[`Storyshots Table With Blue Stripe Style 1`] = `
+<Table
+  table={
+    Object {
+      "header": "BCeID Info",
+      "tableElements": Array [
+        Object {
+          "name": "BCeID",
+          "value": "bobross42",
+        },
+        Object {
+          "name": "Last Name",
+          "value": "Ross",
+        },
+        Object {
+          "name": "First Name",
+          "value": "Bob",
+        },
+        Object {
+          "name": "Email Address",
+          "value": "bob.ross@pbs.com",
```

Comment:
nit: not a blocker but its been suggested to me to use `@example.com` for these types of things so it has no chance of being a real address 😺